muxers/yamux/src/lib.rs

```diff
@@ -80,6 +80,10 @@ where
     fn destroy_outbound(&self, _: Self::OutboundSubstream) {
     }
 
+    unsafe fn prepare_uninitialized_buffer(&self, _: &mut [u8]) -> bool {
+        false
```

Comment:
@twittner Please confirm that `yamux` doesn't read data from the buffer.

Reply:
As far as I can tell, it doesn't.

Reply:
Confirmed.